Wiredcraft/Section/Common/ViewModel/WCBaseViewModel.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  WCBaseViewModel.swift
+//  Wiredcraft
+//
+//  Created by codeLocker on 2020/6/10.
+//  Copyright © 2020 codeLocker. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+class WCBaseViewModel {
+    public let disposeBag = DisposeBag()
```

Comment:
Can you explain me the purpose of the DisposeBag?

Reply:
i think different viewModel may have some subscribe Observable code, and the object that subscribe function return should be put into a DisposeBag to dispose.
i don't want to create DisposeBag for each viewModel, so create a baseViewModel class for each viewModel, and define a DisposeBag in baseViewModel.
so each viewModel can use DisposeBag directly